learn/advanced/faceted_search.md

```diff
@@ -0,0 +1,229 @@
+
+If none of the matching documents have a numeric value for a facet, that facet is not included in the `facetStats` object.
+
+::: note
+Meilisearch ignores numeric strings like `"21"` when computing `facetStats`.
+:::
+
+The following response shows the `facetStats` when searching for `classics`:
+
+```json
+{
+  …
+"facetStats":{
+    "rating":{
+      "min":2.5,
+      "max":4.7
+    }
+  }
+  …
+}  
+```
+
+Since `rating` was the only numeric facet in our example, it is the only facet returned in the `facetStats` object.
+
+## Facet types
+
+### Conjunctive facets
+
+Conjunctive facets use the `AND` logical operator. When users select one or more values for a facet, all returned results must contain the selected facet value(s).
+
+With conjunctive facets, when a user selects `English` from the `language` facet, all returned books must be in English. If the user further narrows down the search by selecting `Fiction` and `Literature` as `genres`, all returned books must be in English and contain both `genres`.
+
+```
+# Filter expression as an array
+["language = English", "genres = Fiction", "genres = Literature"]
+
+# Filter expression as a string
+"language = English AND genres = Fiction AND genres = Literature"
+```
+
+The GIF below shows how the facet count for `genres` updates to only include books that meet **all three conditions**.
+
+![Selecting English books with 'Fiction' and 'Literature' as 'genres' for the books dataset](/faceted-search/conjunctive-factes.gif)
+
+### Disjunctive facets
+
+Disjunctive facets use the `OR` logical operator. They allow users to choose multiple options within a facet, so they don’t have to perform more than one search to find the products they’re looking for.
+
+Let's look at the `books` index from before with disjunctive facets. When the user selects `Fiction` and `Literature` as `genres`, Meilisearch returns all books that are either `Fiction`, `Literature`, or both:
+
+```
+# Filter expression as an array
+[["genres = Fiction", "genres = Literature"]]
+
+# Filter expression as a string
+"genres = Fiction OR genres = Literature"
+```
+
+The GIF below shows the `books` dataset with disjunctive facets. Notice how the facet count for `genres` updates based on the selection.
+
+![Selecting 'Fiction' and 'Literature' as 'genres' for the books dataset](/faceted-search/disjunctive_facets.gif)
+
+### Using conjunctive and disjunctive facets
+
+Let's look at the `books` index with both conjunctive and disjunctive facets. The user selects `English` and `French` from the `language` facet, and then narrows down the search to include `Fiction` and `Literature` as `genres`:
+
+```
+# Filter expression as an array
+[["language = English", "language = French"], "genres = Fiction", "genres = Literature"]
+
+# Filter expression as a string
+"(language = English OR language = French) AND (genres = Fiction AND genres = Literature)"
+```
```

Comment:
I was a bit confused when I reached this section, as I didn't initially understand it was about chaining filter expressions.

I propose holding the user's hand a bit more through each step:

````suggestion
It is possible to create search queries with both conjunctive and disjunctive facets. 

For example, a user might select `English` and `French` from the `language` facet so they can see books written either in English or in French. This query uses an `OR` operator and is a disjunctive facet:

```sql
language = English OR language = French
```

The same user might also be interested in literary fiction books and select `Fiction` and `Literature` as `genres`. Since the user wants a specific combination of genres, their query uses an AND operator:

```sql
genres = Fiction AND genres = Literature

The user can combine these two filter expressions in one by wrapping them in parentheses and using an `AND` operator:

```sql
(language = English OR language = French) AND (genres = Fiction AND genres = Literature)
```
````